G1ANT.Addon.Net/G1ANT.Addon.Net/Commands/MailSmtpCommand.cs

```diff
@@ -68,12 +75,20 @@ public void Execute(Arguments arguments)
             client.Credentials = new System.Net.NetworkCredential(arguments.Login.Value, arguments.Password.Value);
             client.Timeout = (int)arguments.Timeout.Value.TotalMilliseconds;
 
-            string from = arguments.From.Value;
-            string to = arguments.To.Value;
-            string subject = arguments.Subject.Value;
-            string body = arguments.Body.Value;
+            var from = arguments.From.Value;
+            var to = arguments.To.Value;
+            var cc = arguments.Cc.Value.Split(',').ToList(); 
```

Comment:
we also can use ';' to split email addresses

Reply:
The decision was made that we will only use ';' to split the addresses